addons/payment/models/payment_acquirer.py

```diff
@@ -865,8 +865,8 @@ def _cron_post_process_after_done(self):
     @api.model
     def _compute_reference_prefix(self, values):
         if values and values.get('invoice_ids'):
-            many_list = self.resolve_2many_commands('invoice_ids', values['invoice_ids'], fields=['name'])
-            return ','.join(dic['name'] for dic in many_list)
+            invoices = self.new({'invoice_ids': values['invoice_ids']})
+            return ','.join(invoices.invoice_ids.mapped('name'))
```

Comment:
The variable `invoices` is misleading in this context
```suggestion
            invoices = self.new({'invoice_ids': values['invoice_ids']}).invoice_ids
            return ','.join(invoices.mapped('name'))
```